modules/serverless-quarkus-functions-overriding-liveness-readiness.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies
+//
+// * functions/reference/serverless-developing-quarkus-functions.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-quarkus-functions-overriding-liveness-readiness_{context}"]
+= Overriding liveness and readiness probe values
+
+You can override `liveness` and `readiness` probe values for your Quarkus functions. This allows you to modify the timing of health checks performed on the function.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* You have installed the Knative (`kn`) CLI.
+* You have created a function by using `kn func create`.
+
+.Procedure
+
+. Override the `/health/liveness` and `/health/readiness` paths with your own values. You can do this either by changing properties in the function source or by setting the `LIVENESS_URL` and `READINESS_URL` environment variables.
```

Comment:
Here the environment variable are QUARKUS_SMALLRYE_HEALTH_LIVENESS_PATH and QUARKUS_SMALLRYE_HEALTH_READINESS_PATH ...

suggestion:
```suggestion
. Override the `/health/liveness` and `/health/readiness` paths with your own values. You can do this either by changing properties in the function source or by setting the `QUARKUS_SMALLRYE_HEALTH_LIVENESS_PATH` and `QUARKUS_SMALLRYE_HEALTH_READINESS_PATH` environment variables on `func.yaml` file.
```